doc/cli/npm-edit.md

```diff
@@ -3,7 +3,7 @@ npm-edit(1) -- Edit an installed package
 
 ## SYNOPSIS
 
-    npm edit <name>[@<version>]
+    npm edit <pkg>
```

Comment:
For this command, and a few others, making clear that you can fetch a specific version of the package in the command is useful, inasmuch as most people wouldn't even think that was a possibility if they came to the manpage cold.